src/Components/Admin/Scenes/scenecard.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { useHistory } from "react-router-dom";
+import defaultImage from "./defaultImage.svg";
+import Grid from "@material-ui/core/Grid";
+import IconButton from "@material-ui/core/IconButton";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
+
+const useStyles = makeStyles(() => ({
+    card: {
+        fontSize: 16,
+    },
+    cardImage: {
+        width: "90%",
+        maxWidth: 400,
+        marginTop: 16,
+    },
+}));
+
+export default function SceneCard({ key, data }) {
+    const classes = useStyles();
+    const history = useHistory();
+    const [anchorEl, setAnchorEl] = React.useState(null);
+
+    const handleMenuClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleMenuClose = () => {
+        setAnchorEl(null);
+    };
+
+    const handleCardClick = () => {
+        history.push(`/admin/scene/${data.id}`);
```

Comment:
This would go to the embedded A-frame inspector right?

Reply:
You'll need to work with the designers, but we might want some sort of parent screen with information like name, last modified, etc with a button to launch aframe inspector.

Otherwise, we can have all this info in a popup or sorts and just link this to aframe inspector right away. I'm okay with either, comes down to a design decision imo

Reply:
Going to leave this as is, we can change it in a future PR.